### PR TITLE
#184948150 #184948146 #184948151 setup sass and redux and add client readMe file

### PR DESCRIPTION
**Pull request by blackd44**
#### What does this PR do?
- setup sass in the project
- Setup Redux for project states
- Create the client README 

#### Description of Task to be completed?
- Install and setup sass pre-processing on the application to enable the use of both .scss and .css files on the client\
   also add global variables for css
- readme file for the API to include the information below:
  * description
  * installation
  * dependences
- Redux can be used as a data store for any UI layer

#### How should this be manually tested?
after cloning this repository
1. add environment variables as are in `.env.example`
2. run `npm install` to install all dependencies
3. run `npm start` and open in it the browser

#### Any background context you want to provide?
dependencies
- sass
- redux
- react-redux
- @reduxjs/toolkit

#### What are the relevant pivotal tracker/Trello stories?
- [Sass](https://www.pivotaltracker.com/story/show/184948150)
- [Redux]()
- [Read Me]()
#### Screenshots (if appropriate)
- for sass
   Screen Shot 2023-04-18 at 13 35 50
   Screen Shot 2023-04-18 at 13 36 36
- for redux
   Screen Shot 2023-04-18 at 13 44 45

   
#### Questions:
n/a

### CODE REVIEW

**Review by nshutijonathan — APPROVED** [review]
LGTM

File: README.md
Comment (UmuhireJessie) [inline README.md:1]: Add `Authors and acknowledgment` section

**Review by UmuhireJessie — CHANGES_REQUESTED** [review]
See my comment!

**Comment by blackd44** [comment]
> See my comment!

fixed

**Review by edwinnambaje — APPROVED** [review]
LGTM

**Review by UmuhireJessie — APPROVED** [review]

**Pull request merged**